src/popup/components/InputSelectAsset.vue

```diff
@@ -64,7 +64,7 @@ export default {
 
   padding: 2px 12px;
   background-color: rgba(variables.$color-black, 0.3);
-  border-radius: 16px;
+  border-radius: variables.$card-border-radius;
```

Comment:
In my opinion that is an accidental coincidence the value is similar. But the scope is different - this input has nothing to do with the cards.